includes/pages/admin_questions.php

```diff
@@ -153,13 +147,11 @@ function admin_questions()
                     return error('Incomplete call, missing Question ID.', true);
                 }
 
-                $question = DB::selectOne(
-                    'SELECT * FROM `Questions` WHERE `QID`=? LIMIT 1',
-                    [$question_id]
-                );
+                $questions =   Question::where(['id' => $question_id]);
```

Comment:
Using `Question::find($question_id)` directly returns the question or `null`

Reply:
You are right, however because I had built the model based on the old table I was forced to do it like this due to the delete not working correctly due to a lack of an id column.

I will refactor this as it should now be possible to delete normally.